src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerFileReference.cs

```diff
@@ -462,16 +462,15 @@ public bool Equals(AnalyzerReference other)
         {
             if (other != null)
             {
-                return other.Display == this.Display &&
-                       other.FullPath == this.FullPath;
+                return other.FullPath == this.FullPath;
             }
 
             return base.Equals(other);
         }
 
         public override int GetHashCode()
         {
-            return Hash.Combine(this.Display, this.FullPath.GetHashCode());
+            return this.FullPath.GetHashCode();
```

Comment:
>return this.FullPath.GetHashCode(); [](start = 12, length = 35)

It looks like Equals delegates to the base. Should we also combine GetHashCode from the base?

Reply:
Scratch that, it looks like base doesn't have special implementation 

---
In reply to: [179316100](https://github.com/dotnet/roslyn/pull/25884#discussion_r179316100) [](ancestors = 179316100)